src/store/models/network.ts

```diff
@@ -126,6 +126,7 @@ export interface NetworkModel {
   >;
   start: Thunk<NetworkModel, number, StoreInjections, RootModel, Promise<void>>;
   stop: Thunk<NetworkModel, number, StoreInjections, RootModel, Promise<void>>;
+  stopAll: Thunk<NetworkModel, number, StoreInjections, RootModel, Promise<void>>;
```

Comment:
You can use `void` instead of `number` for the second generic type parameter so you don't need to pass an unnecessary arg to `stopAll(0)`

Reply:
Thank you. I actually wanted to ask if there is a way to avoid passing a number.